### PR TITLE
fix: cxg_admin comments

### PR DESCRIPTION
**Pull request by MDunitz**
### Reviewers
**Functional:** 
- @maniarathi 

**Readability:** 

---

## Changes
- modify collection ownership transfer script

### CODE REVIEW

File: scripts/cxg_admin.py [hunk 1]
```diff
@@ -100,12 +102,15 @@ def update_collection_owner(ctx, collection_uuid, new_owner):
 @click.pass_context
 def transfer_collections(ctx, curr_owner, new_owner):
     """Transfer all collections owned by the curr_owner to the new_owner. You must first SSH into the target
-    deployment using `make db/tunnel` before running."""
+    deployment using `make db/tunnel` before running.
+    To run (from repo root)
+    ./scripts/cxg_admin.py --deployment prod transfer-collections $CURR_OWNER_ID $NEW_OWNER_ID
```
Comment (maniarathi) [inline scripts/cxg_admin.py:107]: Might add a note on how to get these IDs? Probably needs the help of an engineer to look in the database? Or look at your own API response?

File: scripts/cxg_admin.py [hunk 2]
```diff
@@ -100,12 +102,15 @@ def update_collection_owner(ctx, collection_uuid, new_owner):
...
+    """
 
     with db_session_manager() as session:
-        collections = session.query(DbCollection).filter(DbCollection.owner == curr_owner)
+        collections = session.query(DbCollection).filter(DbCollection.owner == curr_owner).all()
         new_owner_collections_count = len(session.query(DbCollection).filter(DbCollection.owner == new_owner).all())
-        if collections.all():
+        if len(collections) > 0:
```
Comment (maniarathi) [inline scripts/cxg_admin.py:113]:
```suggestion
        if len(collections):
```

**Review by maniarathi — APPROVED** [review]
Just a couple minor things/nits then LGTM

File: scripts/cxg_admin.py [hunk 3]
```diff
@@ -100,12 +102,15 @@ def update_collection_owner(ctx, collection_uuid, new_owner):
 @click.pass_context
 def transfer_collections(ctx, curr_owner, new_owner):
     """Transfer all collections owned by the curr_owner to the new_owner. You must first SSH into the target
-    deployment using `make db/tunnel` before running."""
+    deployment using `make db/tunnel` before running.
```
Comment (MDunitz) [inline scripts/cxg_admin.py:105]:
```suggestion
    deployment using `make db/tunnel` before running.
    Retrieve user ids from auth0 before running or ping an engineer on the team to check the id of the owner in the database
```

**Pull request merged**